src/styles/variables.ts

```diff
@@ -0,0 +1,4 @@
+// Colors
+export const COLOR_WHITE = '#FFFFFF';
+export const COLOR_DARK_PRIMARY = '#001E2B';
+export const COLOR_DARK_SECONDARY = '#023430';
```

Comment:
So these are all theme values, and I would lean towards using those instead. In general, If you go to [this file](https://github.com/10gen/flora/blob/main/tokens.js) and just search for the color, you can find the corresponding theme value.

Reply:
I totally agree, I wanted to use directly from theme values too but I just couldn't find anywhere to reference them lmao. I will change these to theme values.